src/Components/CreateImageWizardV2/steps/TargetEnvironment/Aws/index.tsx

```diff
@@ -33,9 +33,9 @@ export type AwsShareMethod = 'manual' | 'sources';
 
 // The Sources API only defines a V1ListSourceResponseItem[] type
 export type V1ListSourceResponseItem = {
-  id?: string;
-  name?: string;
-  source_type_id?: string;
+  id?: string | '';
+  name?: string | '';
+  source_type_id?: string | '';
```

Comment:
Empty string is still a string so instead of `string | string` there could be only `string` :)

```suggestion
  id?: string;
  name?: string;
  source_type_id?: string;
```